chef/data_bags/crowbar/migrate/nova/033_add_configdrive.rb

```diff
@@ -0,0 +1,13 @@
+def upgrade ta, td, a, d
+  if a['force_config_drive'].nil?
+    a['force_config_drive'] = ta['force_config_drive']
+  end
+  return a, d
+end
+
+def downgrade ta, td, a, d
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `td`. If it's necessary, use `_` or `_td` as an argument name to indicate that it won't be used.<br>Style/MethodDefParentheses: Use def with parentheses when there are parameters.